tests/agent_equipment/test_launcher.py

```
@@ -0,0 +1,830 @@
+        self,
+        *,
+        fake_bin: Path | None = None,
+        cwd: Path | None = None,
+        environment: dict[str, str] | None = None,
+        launcher: Path | None = None,
+        timeout: float | None = None,
+    ) -> subprocess.CompletedProcess[str]:
+        path = os.environ.get("PATH", "")
+        if fake_bin is not None:
+            path = f"{fake_bin}{os.pathsep}{path}"
+        return subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)